libs/utils/utils.js

```diff
@@ -137,9 +138,9 @@ export const [setConfig, getConfig] = (() => {
   ];
 })();
 
-export function getMetadata(name) {
+export function getMetadata(name, doc = document) {
```

Comment:
@auniverseaway Can you check out this piece of feedback that I gave in this PR? Saugat is loading a html file and wants to select the metadata from it. I suggested that he expand this method in utils.js, so that one can pass any document to read metadata from it. 
The only "problem", is we're extending this code for a use case we currently only have in one place, which I know is usually a bit of a no go. 
Good or bad?